src/renderer/components/AuthenticatedWrapper.vue

```diff
@@ -75,17 +64,62 @@ export default {
     };
   },
 
-  computed: {
-    ...mapGetters([
-      'menuCollapsed',
-      'menuToggleable',
-    ]),
+  methods: {
+    connectWebsocket() {
+      const index = Vue._installedPlugins.indexOf(VueNativeSock);
+
+      if (index > -1) {
+        this.$disconnect();
+        Vue._installedPlugins.splice(index, 1);
```

Comment:
Interesting solution of removing from installed plugins. Will anything prevent the previous instance from being garbage collected?